docs/develop/api-reference/http-api.md

```diff
@@ -21,146 +21,151 @@ This documentation does not describe how to implement HTTP request syntax as tha
+
+|   | AWS Region     | Endpoints                                                    |
+|---|----------------|--------------------------------------------------------------|
+| 1 | us-west-2      | https://api.cache.cell-4-us-west-2-1.prod.a.momentohq.com    |
+| 2 | us-east-1      | https://api.cache.cell-us-east-1-1.prod.a.momentohq.com      |
+| 3 | ap-northeast-1 | https://api.cache.cell-ap-northeast-1-1.prod.a.momentohq.com |
+| 4 | eu-west-1      | https://api.cache.cell-1-eu-west-1-1.prod.a.momentohq.com    |
+| 5 | ap-south-1     | https://api.cache.cell-1-ap-south-1-1.prod.a.momentohq.com   |
 
-|   | AWS Region     | Endpoints                                            |
-|---|----------------|------------------------------------------------------|
-| 1 | us-west-2      | api.cache.cell-4-us-west-2-1.prod.a.momentohq.com    |
-| 2 | us-east-1      | api.cache.cell-us-east-1-1.prod.a.momentohq.com      |
-| 3 | ap-northeast-1 | api.cache.cell-ap-northeast-1-1.prod.a.momentohq.com |
-| 4 | eu-west-1      | api.cache.cell-1-eu-west-1-1.prod.a.momentohq.com    |
-| 5 | ap-south-1     | api.cache.cell-1-ap-south-1-1.prod.a.momentohq.com   |
 
 If you do not see a Region you require, let’s discuss. Please contact [Support](mailto:support@momentohq.com).
 
-## Get Method
+## Cache
+
+### Get Value
+
 Gets a scalar value from a cache.
-### Request
+
+#### Request
 
 - Path: /cache/{cacheName}
 - HTTP Method: GET
 
-#### Path Parameters
+##### Path Parameters
 
 | Parameter&nbsp;name | Required? | Type            | Description                          |
 |---------------------|-----------|-----------------|--------------------------------------|
 | cacheName           | yes       | URL-safe string | The name of the cache to operate on. |
 
-#### Query Parameters
+##### Query Parameters
 
 | Parameter&nbsp;name | Required? | Type                                         | Description                                                                                           |
 |---------------------|-----------|----------------------------------------------|-------------------------------------------------------------------------------------------------------|
 | key                 | no**      | URL-safe&nbsp;string                         | The key to retrieve from the cache.                                                                   |
 | key_base64          | no**      | Base64&nbsp;URL-encoded&nbsp;binary&nbsp;key | The key to retrieve from the cache.                                                                   |
-| token               | no***     | URL-safe string                              | The Momento auth token, in string format, to be used for authentication/authorization of the request. |
+| token               | no***     | URL-safe string                              | The Momento API token, in string format, to be used for authentication/authorization of the request. |
```

Comment:
I don't have a strong preference for how we refer to these tokens in the docs, but there are several places in the SDK code where they are referred to as Auth tokens (e.g. `generateAuthToken`, `refreshAuthToken`, `AuthClient`, `MOMENTO_AUTH_TOKEN`, `CredentialProvider.fromString({authToken: ...})`.

There was a time when people started flipping the docs over to say API tokens, and we had this exact conversation, and people decided it was better to make the docs consistent with the code.

But like I said, I don't feel that strongly about it... if you do feel strongly, just be aware that if you start changing references like this to say "API" we're intentionally choosing to have 2 different ways of referring to them, at least until we decide to do a 2.0 of all of the SDKs.

Reply:
I don't feel strongly, I'll make it say auth token.